### PR TITLE
Visual novel editor 0.1v - only text

### PR DESCRIPTION
**Pull request by namse**
# What this editor supports
- CRUD Sequence
- CRU Line of dialogue
- Auto save into http://github.com/namseent/luda-editor-client

### CODE REVIEW

File: luda-editor/client/src/pages/sequence_edit_page/components/mod.rs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// pub mod image_browser;
+// pub mod property_editor;
+// pub mod timeline;
+// pub mod wysiwyg_editor;
```
Comment (namse) [inline luda-editor/client/src/pages/sequence_edit_page/components/mod.rs:4]: Will used next time

Comment (namse) [reply luda-editor/client/src/pages/sequence_edit_page/components/mod.rs:4]: Don't spend your time to review components and wysiwyg editor this time.

File: luda-editor/crdt/src/tests/yrs_test.rs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    {
+        let mut txn = doc.transact();
+        let root = txn.get_text("root-type-name");
+        root.push(&mut txn, "hello world");
+    };
+    let encoded_update_1 = doc.encode_state_as_update_v2(
+        &StateVector::decode_v2(&remote_encoded_state_vector.as_slice()).unwrap(),
+    );
+    println!("{:?}", encoded_update_1);
```
Comment (bigfoodK) [inline luda-editor/crdt/src/tests/yrs_test.rs:23]: Is this for debug or test?

File: luda-editor/core/src/storage/github_storage/mod.rs [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+    //             }
+    //         }
+    //     });
+
+    //     let github_storage = GithubStorage::new(mock_client, "master".to_string());
+
+    //     let result = github_storage.start_sync(&mut history_system).await;
+    //     assert!(result.is_ok());
+    //     assert_eq!(call_count.load(std::sync::atomic::Ordering::Relaxed), 5);
+    // }
```
[the remark this answers is not in the document]
Comment (namse) [reply luda-editor/core/src/storage/github_storage/mod.rs:393]: Yes I commented it to use next time. architecture changed so I couldn't make same test.

File: luda-editor/core/src/github_client/methods/put_repository_content.rs [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+//     },
+//     "payload": {
+//       "type": [
+//         "string",
+//         "null"
+//       ]
+//     }
+//   }
+// }
+// }
```
[the remark this answers is not in the document]
Comment (namse) [reply luda-editor/core/src/github_client/methods/put_repository_content.rs]: I don't think so. Will remove

**Comment by namse** [comment]
WasmUnknownWeb, WasmWindowsElectron, start and build tested.

**Review by bigfoodK — APPROVED** [review]

**Pull request merged**